File: task-03/python/task_3.py
n=int(input("Enter Number U want to find Prime Numbers till:-"))
print("Prime Numbers Are:-")

for x in range(2,n):
    
    for i in range(2,x//2+1):#we need to only check till half the number to find if it has a factor or not 
        if x%i==0:
    
            break
    else:
        print(x)

   

File: task-03/c++/task3.cpp
#include <iostream>
using namespace std;

bool checkPrime(int n){
    for (int i=2; i<n; i++){
        if(n%i==0){
            return false;
        }
    }
    return true;
}

int main(){
    int num;
    cout << "Enter Value ";
    cin >> num;
    int i;
    if ((num==1)||(num==0)){
        cout<<"No Prime in Range\n";
        return 0;
    }
    for (i=2;i<num;i++){
        if (checkPrime(i)){
            cout<<i<<" ";
        }

    }
    cout<<"\n";

}